LocationServices/LocationServices.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,32 +1,69 @@
 {
+  "originHash" : "33fa476c44ec13ba9dfb107bc55a239c88b17c3ebefc9a581f9498da82ded5fd",
   "pins" : [
     {
-      "identity" : "aws-sdk-ios-spm",
+      "identity" : "aws-crt-swift",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/aws-amplify/aws-sdk-ios-spm",
+      "location" : "https://github.com/awslabs/aws-crt-swift",
       "state" : {
-        "revision" : "477ba09977beda6f259f225f2e4eca37f0a9de05",
-        "version" : "2.30.1"
+        "branch" : "mqtt_test_app",
+        "revision" : "e7e603d3aadd1749b3e5cc554832ab337fdfd8f3"
```

Comment:
It's a dependency of aws-sdk-swift, so I think it's ok (and necessary) for us to be using it in general:
https://github.com/awslabs/aws-sdk-swift/blob/main/Package.swift#L427

However, the question still remains on why we're using the mqtt_test_app branch instead of the latest released version.

Reply:
mqtt_test_app is not merged in main branch and There is no mqtt implementation in main. I asked on this branch PR when it is getting merged but didn't get any reply. I couldn't find any mqtt implementation for iOS other than c/c++ SDK for mqtt. This branch is doing the same calling mqtt c sdk via middleware classes

Reply:
I synced with the developers, we can stick with the mqtt_test_app branch for now. We'll just need to revisit at some point after it gets merged into mainline.